### DISCUSSION TITLE
Test for Debugger

### DISCUSSION THREAD

**Discussion by winst0niuss** [getgauge/taiko#2630] [General]
I'm testing a debugger in an IDE one application for developers(looks like a copy Vcode).
I hit the breakpoint, start debugging, => the project page opens and stops at the breakpoint. The page is frozen.
As expected for debugging, the page is frozen. I can't click anything on it.

 When manually testing - I can return to the debugger tab and see that debugging really stopped at the breakpoint, as well as turn off debug mode, after which the project page is unfrozen.

But when working with Taiko - after freezing the page, I can't get back to the debugger page and none of the actions work. How do I get to the debugger page? Is there any forced option to open another tab?

ps The test does  fail on timeout error

**Comment by NivedhaSenthil** [comment]
Sorry I did not understand the flow correctly, can you please provide steps for me try this debug scenario ?

**Comment by winst0niuss** [comment]
Thank you. Not relevant, we managed to solve the problem on the project side
